### PR TITLE
[AIRFLOW-4442] fix hive_tblproperties in HiveToDruidTransfer

### PR DESCRIPTION
**Pull request by RosterIn**
HiveToDruidTransfer execute() func uses self.hive_tblproperties.items()
if self.hive_tblproperties is None then this raise error.
Modify it to empty dict if it's None.

Make sure you have checked _all_ steps below.

### Jira

- [ ] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW-4442) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-4442
  - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a Jira issue.
  - In case you are proposing a fundamental code change, you need to create an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)).
  - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).

### Description

- [ ] Here are some details about my PR, including screenshots of any UI changes:

### Tests

- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [ ] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public functions and the classes in the PR contain docstrings that explain what it does
  - If you implement backwards incompatible changes, please leave a note in the [Updating.md](https://github.com/apache/airflow/blob/master/UPDATING.md) so we can assign it to a appropriate release

### Code Quality

- [ ] Passes `flake8`

### CODE REVIEW

**Comment by RosterIn** [comment]
waiting for review :)

File: airflow/operators/hive_to_druid.py
Comment (feluelle) [inline airflow/operators/hive_to_druid.py]:
```suggestion
        self.hive_tblproperties = hive_tblproperties or {}
```

**Review by feluelle — APPROVED** [review]
Good catch. LGTM 👍 (I personally prefer `{}` to `dict()`)

File: airflow/operators/hive_to_druid.py
Comment (RosterIn) [reply airflow/operators/hive_to_druid.py]: @feluelle apparently it's not just a matter or preference it's also a matter of performance. `{}` is much better:
https://stackoverflow.com/questions/17097985/dict-vs-in-python-which-is-better/17098017
https://stackoverflow.com/questions/5790860/and-vs-list-and-dict-which-is-better

**Comment by RosterIn** [comment]
Not sure why the kubernetes test failed.
@feluelle can you help with this?

**Comment by feluelle** [comment]
To me that looks unrelated to your code changes.

So you can either rebase onto master to rerun the tests again or wait for a committer to rerun the failed tests :)

**Comment by RosterIn** [comment]
@Fokko @kaxil mind take a look?

**Comment by Fokko** [comment]
@RosterIn I've restarted the failing CI

**Review by Fokko — APPROVED** [review]

**Comment by Fokko** [comment]
Thanks @RosterIn !

**Comment by Fokko** [comment]
@ashb @kaxil Are we still using the test branch? I've cherry-picked the commit to `v1-10-test`.

**Comment by ashb** [comment]
@Fokko We've brought it back again -- now that we have remove Py2 from master it is more likely that cherry-picks might fail, so pulling fixes in to there, and then once they pass tests merging -test on to -stable is what I have been doing. (i.e. v1-10-test may be rebase/force-pushed)

**Pull request merged**
